plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -104,7 +104,7 @@
    <extension
          point="org.eclipse.wst.server.core.publishTasks">
       <publishTask
-            class="com.google.cloud.tools.eclipse.appengine.localserver.server.CloudSdkPublishTaskDelegate"
+            class="com.google.cloud.tools.eclipse.appengine.localserver.server.LocalAppEnginePublishTaskDelegate"
```

Comment:
So is the problem actually the stubs or is it this? Or is it both of them together?